sbndqm/dqmAnalysis/CRT/SBND/BernCRTdqmSBND_module.cc

```diff
@@ -413,94 +487,197 @@ if(!fragmentHandle.isValid() || fragmentHandle->size() == 0)
-    sbndaq::sendMetric("CRT_board", readout_number_str, "MaxADCChannel", maxindex + 32 * mac5, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "Flag", thisflag, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "baseline", baseline, 0, artdaq::MetricMode::Average);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "TS0", ts0, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "TS1", ts1, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "Deadtime", deadtime, 0, artdaq::MetricMode::Minimum);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "MissingT0", missingT0, 0, artdaq::MetricMode::Maximum);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "MissingT1", missingT1, 0, artdaq::MetricMode::Maximum);
+     * We note that the level here is used as a threshold; we send any value of the metric above the level. For the CRT DQM, all levels = 0.
+     */
+
+    sbndaq::sendMetric("CRT_board", mac5_str, "Flag", thisflag, 0, artdaq::MetricMode::LastPoint);
+    sbndaq::sendMetric("CRT_board", mac5_str, "TS0", static_cast<int>(ts0), 0, artdaq::MetricMode::LastPoint);
+    sbndaq::sendMetric("CRT_board", mac5_str, "TS1", static_cast<int>(ts1), 0, artdaq::MetricMode::LastPoint);
+    sbndaq::sendMetric("CRT_board", mac5_str, "MissingT0", missingT0[mac5], 0, artdaq::MetricMode::Maximum);
+    sbndaq::sendMetric("CRT_board", mac5_str, "MissingT1", missingT1[mac5], 0, artdaq::MetricMode::Maximum);
  
     //only send clockdrift info when it makes sense to do so; that is, for T0 reset events.
-    if(isTS0 && isTS0good) {sbndaq::sendMetric("CRT_board", readout_number_str, "T0clockdrift", ts0 - 1e9, 0, artdaq::MetricMode::LastPoint);}
-    if(isTS1 && isTS1good) {sbndaq::sendMetric("CRT_board", readout_number_str, "T1clockdrift", ts1 - 1e9, 0, artdaq::MetricMode::LastPoint);}
+    if(isTS0 && isTS0good) {sbndaq::sendMetric("CRT_board", mac5_str, "T0clockdrift", static_cast<int>(ts0) - 1e9, 0, artdaq::MetricMode::LastPoint);}
+    if(isTS1 && isTS1good) {sbndaq::sendMetric("CRT_board", mac5_str, "T1clockdrift", ts1 - 1e9, 0, artdaq::MetricMode::LastPoint);}
 
     //Sychronization Metrics
-    sbndaq::sendMetric("CRT_board", readout_number_str, "earlysynch", earlysynch, 0, artdaq::MetricMode::Average);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "latesynch", latesynch, 0, artdaq::MetricMode::Average);
+    sbndaq::sendMetric("CRT_board", mac5_str, "earlysynch", earlysynch, 0, artdaq::MetricMode::Average);
+    sbndaq::sendMetric("CRT_board", mac5_str, "latesynch", latesynch, 0, artdaq::MetricMode::Average);
 
     // Flag 3 Hits (Board Level)
-    sbndaq::sendMetric("CRT_board", readout_number_str, "Flag3Hit", flag3hit, 0, artdaq::MetricMode::Average);  
+    sbndaq::sendMetric("CRT_board", mac5_str, "Flag3Hit", flag3hit, 0, artdaq::MetricMode::Average);  
```

Comment:
Does this make sense to send? It can only take value 1/0 does it tell us anything in average form that we don't already get from the reset / missing and readout rate counters?

Reply:
Hmm.. The way I see it it's bounded upper by 32 not 1?

Reply:
```
auto currflag = hit.flags;
for(int i = 0; i<32; i++) {
  if (currflag == 3) {
    flag3hit++;
  }
//...
}
```
Could be there's a physics reason why `flag3hit` is 0 or 1.. 🤔

Reply:
Actually that's even more mad than I realised... it's incrementing a module level variable in a channel loop. So it can actually take any value from 0 to 32 :-1: 

Let's remove this completely. The idea behind it is implemented properly in `chReadoutRate` and `readoutRate`